gradle/dependencies.gradle

```diff
@@ -51,6 +51,7 @@ versions += [
   apacheda: "1.0.0",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
+  avro4sVersion: "1.8.3",
```

Comment:
I think we do not need avro4sVersion any more? Same as line 86 here.

Reply:
removed ..